ui/src/views/Documentation/components/Anchor/index.jsx

```diff
@@ -9,29 +9,33 @@ const useStyles = withStyles(theme => ({
     ...theme.mixins.link,
   },
 }));
+// Ref: https://material-ui.com/guides/composition/#caveat-with-refs
+// eslint-disable-next-line react/display-name
```

Comment:
To fix the lint rule, I should use a normal `function` as the callback but I am unable to do so as we have another lint rule which requires arrow functions as callbacks and not normal functions.